src/cmdstan/command.hpp

```diff
@@ -338,7 +336,7 @@ int command(int argc, const char *argv[]) {
           save_single_paths, refresh, interrupt, logger, init_writer,
           sample_writers[0], diagnostic_json_writers[0]);
     } else {
-      auto ofs = std::make_unique<std::ofstream>(output_base + ".csv");
+      auto ofs = std::make_unique<std::ofstream>(output_file);
```

Comment:
using make_filenames and taking the first element should make the behavior more consistent with other methods